yafti/cli/main.py

```diff
@@ -0,0 +1,121 @@
+import logging
+import subprocess
+import sys
+from pprint import pprint
+from typing import Annotated
+
+import typer
+
+import yafti.setup  # noqa
+from yafti import __version__, log
+from yafti.app import Yafti
+
+CONTEXT_SETTINGS = dict(help_option_names=["-h", "--help"])
+app = typer.Typer(context_settings=CONTEXT_SETTINGS, help="yafti command line tool")
+
+
+@app.callback()
+def callback(ctx: typer.Context) -> None:
+    """callback"""
+    pass
+
+
+@app.command()
+def version(ctx: typer.Context) -> None:
+    """
+    Show yafti version
+    """
+    typer.echo(f"Platform:         {sys.platform}")
+    typer.secho(f"Python Version:   {sys.version}")
+    typer.secho(f"Yafti Version:    {__version__}")
```

Comment:
https://github.com/python-poetry/poetry/issues/273#issuecomment-401983643

Reply:
Actually, that is not the best way, Py3 has `importlib.metadata.version("yafti")` which will do what you want